core/modules/widgets/attributes/macro.js

```diff
@@ -0,0 +1,44 @@
+/*\
+title: $:/core/modules/widgets/attributes/macro.js
+type: application/javascript
+module-type: attributevalue
+
+An attribute value acquired via macro expansion.
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+var Formulas = require("$:/plugins/ebalster/formula/compile.js");
+var Operands = require("$:/plugins/ebalster/formula/operands.js");
+
```

Comment:
This is not allowed in a PR for the core!!

Reply:
Just wanted to test your PR, but this breaks it.

Reply:
It may be an oversight?

Reply:
Oversight on my part, fixed in a new commit.

Something that threw me is that modifying these files doesn't result in any changes if I build the empty edition.  Should I be invoking the compiler elsewhere?